### PR TITLE
Fix documentcard theming

### PR DESCRIPTION
**Pull request by joschect**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.scss [hunk 1]
```diff
@@ -162,7 +162,7 @@ $ms-DocumentCardActivity-personaTextGutter: 8px;
     top: -1px;
     right: -1px;
     bottom: -1px;
-    border: 2px solid #fafafa; // Match the background of the card
+    border: 2px solid $ms-color-white; // Match the background of the card
```
Comment (phkuo) [inline packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.scss]: do we need the border?

File: packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.scss [hunk 2]
```diff
@@ -103,7 +103,7 @@ $ms-DocumentCardActions-verticalPadding: 4px;
 
   :global(.ms-Button:hover) {
     :global(.ms-Button-icon) {
-      color: #1174c3;
+      color: $ms-color-neutralLight;
```
Comment (phkuo) [inline packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.scss]: prefer to use one of the button semantic slots

Comment (joschect) [reply packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.scss]: Which one?

File: packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.scss (hunk 1 above)
Comment (joschect) [reply packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.scss]: I do not know.

Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.scss]: neutralLighterAlt?

Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.scss]: I think it would be wise to keep a visible border if there was one.

**Pull request merged**